modin/pandas/test/dataframe/test_join_sort.py

```diff
@@ -305,13 +305,23 @@ def test_merge(test_data, test_data2):
         )
         df_equals(modin_result, pandas_result)
 
-    # Named Series promoted to DF
-    s = pd.Series(frame_data2.get("col1"))
-    with pytest.raises(ValueError):
-        modin_df.merge(s)
+    # Cannot merge a Series without a name
```

Comment:
Does this mean we previously allowed merging with a nameless Modin series (incorrectly) but now we do not (correctly)?

Reply:
I think pandas changed their behavior, so it no longer errors.